src/contracts/BMath.sol

```diff
@@ -7,15 +7,19 @@ import {BNum} from './BNum.sol';
 
 contract BMath is BBronze, BConst, BNum {
   /**
-   *
-   *    calcSpotPrice
-   *    sP = spotPrice
-   *    bI = tokenBalanceIn                ( bI / wI )         1
-   *    bO = tokenBalanceOut         sP =  -----------  *  ----------
-   *    wI = tokenWeightIn                 ( bO / wO )     ( 1 - sF )
-   *    wO = tokenWeightOut
-   *    sF = swapFee
-   *
+   * @notice Calculate the spot price of a swap
```

Comment:
* of a token in terms of another one?

Reply:
here `swap` is incorrect bc if you swap, then is not "spot-price" because you're modifying the balances